### DISCUSSION TITLE
Nightly 108 update hides bottom toolbar icons.

### DISCUSSION THREAD

**Discussion by Pizzapops** [Aris-t2/CustomCSSforFx#499] [General]
After updating My Nightlys to **108**, my bottom toolbar was blank. After several tries to revive the icons, I found the answer. If a **Flexible Space** has been used in the bottom toolbar, it takes over the entire space and every thing else is hidden. Using Customize Toolbar and drag all Flexible Spaces out of the toolbar. After clicking the DONE button, all icons and single should reappear aligned to the left side.

**Comment by Aris-t2** [comment]
I could not reproduce this. Any ideas? Or is this not about `addonbar_move_bookmarks_toolbar_to_bottom.css`.

![image](https://user-images.githubusercontent.com/6013569/197203321-fd8e602b-b1ee-466b-802e-e0242a8d8b5d.png)
![image](https://user-images.githubusercontent.com/6013569/197203351-936fcef3-1a49-4c20-943b-18036b56db3e.png)

**Comment by Pizzapops** [comment]
Aris, 
The problem may be that I am using _**addonbar.uc.js**_ and it  includes some minor mods. I preferred that because it included the single space, a vertical line separator and the variable space. I have had no problem with FxESR, Fx106, FxBeta, Waterfox5 or TOR. All variations of Fx108  have the problem but only with the bottom addon bar. I will compare my **_addonbar.uc.js_** to my **_addonbar_vertical.uc.js_** to see if that is where the error is. 

![FX108_2022-10-21](https://user-images.githubusercontent.com/33039961/197240081-f19cadbf-d9b2-4ba1-9afe-73e3ea95967d.JPG)

**Reply by Aris-t2** [reply]
Try this:
`addonbar.uc.js` > line 66 > `min-width: unset !important;\`

```js
		  #addonbar toolbarpaletteitem[place=toolbar][id^=wrapper-customizableui-special-spring],\
		  #addonbar toolbarspring {\
			-moz-box-flex: 1 !important;\
			min-width: 100% !important;\
			width: unset !important;\
			max-width: unset !important;\
		  }\
```
--->
```js
		  #addonbar toolbarpaletteitem[place=toolbar][id^=wrapper-customizableui-special-spring],\
		  #addonbar toolbarspring {\
			-moz-box-flex: 1 !important;\
			min-width: unset !important;\
			width: unset !important;\
			max-width: unset !important;\
		  }\
```

**Comment by Pizzapops** [comment]
Aris,

The above did not help. So I did some testing. Results:

No **_userChrome.css_** & no **_userContent.css_** but still **_userChrome.js_** (18).  Addonbar  variable space still over-expands.

Deactivate all scripts except current **_addonbar.uc.js_**. Same results.

My next step is to test my extensions to see if one of them is the problem. I doubt it because is should show up in the other versions of Fx. 
It may have something do with the non-movable extensions icon (top-right) that appeared about the time 108 was released. Thanks Devs!

If I can find the time, I may try new portable Fx108 with a clean profile to check results.

**Reply by Aris-t2** [reply]
Not sure why it is not working for you. I tested the above code change (which is part of the [current addonbar.uc.js](https://github.com/Aris-t2/CustomJSforFx/blob/master/scripts/addonbar.uc.js)) on Fx 102ESR, 106 and 108 Nightly with and without flexible spaces and "addonbar" behaved as expected. Maybe there is another conflict.

The extensions button does not cause the issue. You can hide it, if you like, by enabling this one here: https://github.com/Aris-t2/CustomCSSforFx/blob/master/classic/userChrome.css#L329

**Comment by Aris-t2** [comment]
Is this issue still present?
